### PR TITLE
feat: add tpms pressure sensors

### PR DESCRIPTION
**Pull request by InTheDaylight14**
Mentioned in https://github.com/alandtse/tesla/issues/294

closes https://github.com/alandtse/tesla/issues/160

PR adds pressure sensors for the four TPMS tire sensors.

### CODE REVIEW

File: custom_components/tesla_custom/sensor.py [hunk 1]
```diff
@@ -497,3 +507,50 @@ def native_value(self) -> Optional[datetime]:
...
+        self._attr_device_class = SensorDeviceClass.PRESSURE
+        self._attr_state_class = SensorStateClass.MEASUREMENT
+        self._attr_native_unit_of_measurement = PRESSURE_BAR
+        self._attr_icon = "mdi:gauge-full"
+        # self._value: Optional[datetime] = None
+
+    @property
+    def native_value(self) -> float:
+        """Return TPMS Pressure."""
+        if self._tpms_sensor == "TPMS front left":
```
Comment (alandtse) [inline custom_components/tesla_custom/sensor.py:534]: This use of a string compare is brittle particularly when it's defined above as a constant. Why not replace the list with a dict with the key being the text string and the value being the attribute we want found?

That way:
```python
    return getattr(self._car, "TPMS_SENSORS.get(self._tpms_sensor))
```

Comment (alandtse) [reply custom_components/tesla_custom/sensor.py:534]: We can also add a default case where if there is no attribute, we return None or something.

Comment (InTheDaylight14) [reply custom_components/tesla_custom/sensor.py:534]: Very cool, I was looking for a way to do that! Thanks

File: custom_components/tesla_custom/sensor.py [hunk 2]
```diff
@@ -19,6 +19,7 @@
     SPEED_MILES_PER_HOUR,
     TEMP_CELSIUS,
     TIME_HOURS,
+    PRESSURE_BAR,
```
Comment (alandtse) [inline custom_components/tesla_custom/sensor.py:22]: Black should be alphabetizing this. Are you sure you're running it?

**Comment by FalconUK** [comment]
Great to see these added. A small point, and it's a picky one, but I think the friendly name should have TPMS all in upper case, rather than just the T.

**Pull request merged**